string controller_name
string[] dof_name
string[] dof_unit
float64[] limit_lower
float64[] limit_upper
string[] gain_name
float64[] kp
float64[] kd
